megamek/src/megamek/common/templates/InfantryTROView.java

```diff
@@ -226,10 +226,8 @@ private void addAugmentationNotes(List<String> notes) {
         }
         if (!options.isEmpty()) {
             notes.add(Messages.getString("TROView.InfantryNote.Augmented"));
-            options.forEach(o -> {
-                notes.add(o.getDisplayableName().replaceAll("\\s+\\(Not Implemn?ented\\)", "") + ": "
-                        + o.getDescription().replaceAll("See IO.*", ""));
-            });
+            options.forEach(o -> notes.add(o.getDisplayableName().replaceAll("\\s+\\(Not Implemented\\)", "") + ": "
```

Comment:
Yikes.

Reply:
I was lazy the first time around and accounted for the spelling error rather that fixing it.